### PR TITLE
Fix missing and wrong license

### PR DESCRIPTION
**Pull request by menghanl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,9 @@ before_install:
   - go get -u golang.org/x/tools/cmd/goimports github.com/axw/gocov/gocov github.com/mattn/goveralls golang.org/x/tools/cmd/cover
 
 script:
-  - '! gofmt -s -d -l . 2>&1 | read'
-  - '! goimports -l . | read'
+  - '! git ls-files "*.go" | xargs grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" | tee /dev/stderr | read'
+  - '! gofmt -s -d -l . 2>&1 | tee /dev/stderr | read'
+  - '! goimports -l . | tee /dev/stderr | read'
```
Comment (dfawley) [inline .travis.yml]: 2>&1

File: .travis.yml [hunk 2]
```diff
@@ -12,8 +12,9 @@ before_install:
   - go get -u golang.org/x/tools/cmd/goimports github.com/axw/gocov/gocov github.com/mattn/goveralls golang.org/x/tools/cmd/cover
 
 script:
-  - '! gofmt -s -d -l . 2>&1 | read'
-  - '! goimports -l . | read'
+  - '! git ls-files "*.go" | xargs grep -L "\(Copyright [0-9]\{4,\} gRPC authors\)\|DO NOT EDIT" | tee /dev/stderr | read'
```
Comment (dfawley) [inline .travis.yml]: 2>&1 with grep

Comment (dfawley) [inline .travis.yml]:
```
set -o pipefail
...
git ls-files ..... | (! read)
```
just to be paranoid

Comment (menghanl) [inline .travis.yml]: Done.

Comment (menghanl) [inline .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (menghanl) [inline .travis.yml]: Done.

**Pull request merged**